easybuild/framework/easyblock.py

```diff
@@ -332,7 +337,7 @@ def fetch_patches(self, list_of_patches, extension=False, checksums=None):
                         patchspec['copy'] = suff
                     else:
                         patchspec['sourcepath'] = suff
-                if level:
+                if level is not None:
```

Comment:
yes, try the test cases with and without this line if you're curious, 

when setting patch level to 0 this if returns false, but you actually want to set '0' not ignore it.
 (later on the guesser would guess level 4, but I explicitly put level 0, so it shouldn't have guessed.
Maybe the guesser could also be fixed, but the patch file was rather deep, and there was a weird directory structure, so I don't blame it for not guessing correct, that's a different issue altogether)